### PR TITLE
Onboard Slack and Zoom SSO templates

### PR DESCRIPTION
**Pull request by DilshanSenarath**
> Documentation for slack and zoom SSO templates.

### Slack Documentation
3-slack

### Zoom Documentation
2-zoom

### Related Issue
- https://github.com/wso2/product-is/issues/21689
- https://github.com/wso2/product-is/issues/21964

### CODE REVIEW

File: en/includes/guides/authentication/sso-integrations/add-slack-template.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Log in to Slack using the {{ product_name }}
+
+This page guides you through using {{ product_name }} to log in to Slack.
```
Comment (pavinduLakshan) [inline en/includes/guides/authentication/sso-integrations/add-slack-template.md:3]:
```suggestion
This page guides you through setting up {{ product_name }} to add SSO (Single Sign-On) login to a Slack workspace.
```

File: en/includes/guides/authentication/sso-integrations/add-slack-template.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    ![Select slack workspace app in the {{ product_name }}]({{base_path}}/assets/img/guides/authentication/sso-integrations/slack/add-slack-app.png){: width="600" style="border: 0.3px solid lightgrey;"}
+
+4. Click **Create** to complete the registration.
+
+-----
+
+## Configure Slack
+
+1. Follow [Slack's documentation](https://slack.com/intl/en-gb/help/articles/212681477-Sign-in-to-Slack){:target="_blank"} to log in to your workspace.
```
Comment (pavinduLakshan) [inline en/includes/guides/authentication/sso-integrations/add-slack-template.md:34]:
```suggestion
1. Follow [Slack documentation](https://slack.com/intl/en-gb/help/articles/212681477-Sign-in-to-Slack){:target="_blank"} to log in to your workspace.
```

File: en/includes/guides/authentication/sso-integrations/add-slack-template.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    !!! warning "Service provider issuer"
+        If you change the Issuer Name to `Slack Workspace URL` when creating the application, you must also configure it in Slack's settings. You can copy the `Service provider issuer` value from the `Guide` tab under the `Setup {{ product_name }} SSO in Slack` step.
+
+    ![Sample Slack SSO Settings Form]({{base_path}}/assets/img/guides/authentication/sso-integrations/slack/sso-fill-slack.png){: width="600" style="border: 0.3px solid lightgrey;"}
+
+-----
+
+## Try it out
+
+Now, you have successfully configured Slack and {{ product_name }}.
```
Comment (pavinduLakshan) [inline en/includes/guides/authentication/sso-integrations/add-slack-template.md:56]:
```suggestion
Now, you have successfully configured Slack SSO in {{ product_name }}.
```

File: en/includes/guides/authentication/sso-integrations/add-slack-template.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+!!! note
+    When the workspace owner enables SSO for the Slack workspace, existing users will receive an email at their registered email address to bind their {{ product_name }} account. New users will be auto-provisioned during login. For more details, refer to Slack's documentation on [What to expect after SSO is enabled](https://slack.com/intl/en-gb/help/articles/203772216-SAML-single-sign-on#what-to-expect){:target="_blank"} and [Sign in to Slack](https://slack.com/intl/en-gb/help/articles/212681477-Sign-in-to-Slack){:target="_blank"}.
```
Comment (pavinduLakshan) [inline en/includes/guides/authentication/sso-integrations/add-slack-template.md:59]:
```suggestion
    When the workspace owner enables SSO for the Slack workspace, existing users will receive an email to their registered email address to bind their {{ product_name }} account. New users will be auto-provisioned during login. For more details, refer to Slack's documentation on [What to expect after SSO is enabled](https://slack.com/intl/en-gb/help/articles/203772216-SAML-single-sign-on#what-to-expect){:target="_blank"} and [Sign in to Slack](https://slack.com/intl/en-gb/help/articles/212681477-Sign-in-to-Slack){:target="_blank"}.
```

File: en/identity-server/next/mkdocs.yml [hunk 5]
```diff
@@ -376,11 +376,13 @@ nav:
             - Add login to an SPA: guides/authentication/add-login-to-single-page-app.md
             - Add login to a web app: guides/authentication/add-login-to-web-app.md
             - Add login to a mobile app: guides/authentication/add-login-to-mobile-app.md
-            - Add login to saas app:
-              - Add SSO login to saas apps: guides/authentication/sso-integrations/index.md
-              - Google Workspace: guides/authentication/sso-integrations/add-google-workspace-template.md
-              - Salesforce: guides/authentication/sso-integrations/add-salesforce-template.md
-              - Microsoft 365: guides/authentication/sso-integrations/add-microsoft-365-template.md
+          - Add login to saas app:
```
[the remark this answers is not in the document]
Comment (DilshanSenarath) [reply en/identity-server/next/mkdocs.yml]: https://github.com/wso2/docs-is/pull/4893/commits/4331f958e1550c4ea18946cbbc29e70d46d2e07b

File: en/identity-server/next/mkdocs.yml [hunk 6]
```diff
@@ -376,11 +376,13 @@ nav:
...
+            - Add SSO login to saas apps: guides/authentication/sso-integrations/index.md
```
[the remark this answers is not in the document]
Comment (DilshanSenarath) [reply en/identity-server/next/mkdocs.yml]: https://github.com/wso2/docs-is/pull/4893/commits/4331f958e1550c4ea18946cbbc29e70d46d2e07b

File: en/includes/guides/authentication/sso-integrations/add-slack-template.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
+# Log in to Slack using the {{ product_name }}
```
[the remark this answers is not in the document]
Comment (DilshanSenarath) [reply en/includes/guides/authentication/sso-integrations/add-slack-template.md]: https://github.com/wso2/docs-is/pull/4893/commits/4331f958e1550c4ea18946cbbc29e70d46d2e07b

File: en/includes/guides/authentication/sso-integrations/add-slack-template.md (hunk 1 above)
Comment (DilshanSenarath) [reply en/includes/guides/authentication/sso-integrations/add-slack-template.md]: https://github.com/wso2/docs-is/pull/4893/commits/4331f958e1550c4ea18946cbbc29e70d46d2e07b

File: en/includes/guides/authentication/sso-integrations/add-slack-template.md (hunk 2 above)
Comment (DilshanSenarath) [reply en/includes/guides/authentication/sso-integrations/add-slack-template.md]: https://github.com/wso2/docs-is/pull/4893/commits/4331f958e1550c4ea18946cbbc29e70d46d2e07b

File: en/includes/guides/authentication/sso-integrations/add-slack-template.md (hunk 3 above)
Comment (DilshanSenarath) [reply en/includes/guides/authentication/sso-integrations/add-slack-template.md]: https://github.com/wso2/docs-is/pull/4893/commits/4331f958e1550c4ea18946cbbc29e70d46d2e07b

File: en/includes/guides/authentication/sso-integrations/add-slack-template.md (hunk 4 above)
Comment (DilshanSenarath) [reply en/includes/guides/authentication/sso-integrations/add-slack-template.md]: https://github.com/wso2/docs-is/pull/4893/commits/4331f958e1550c4ea18946cbbc29e70d46d2e07b

**Review by pavinduLakshan — DISMISSED** [review]

File: en/includes/guides/authentication/sso-integrations/add-zoom-template.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
+# Log in to Zoom using {{ product_name }}
+
+This page guides you through setting up {{ product_name }} to add SSO (Single Sign-On) login to a Zoom organization.
+
+!!! note
+    Ensure you have a Zoom account and your account supports SSO by referring to the [Zoom documentation](https://support.zoom.com/hc/en/article?id=zm_kb&sysparm_article=KB0060673){:target="_blank"}.
+
+## Create the Zoom Service Provider
```
Comment (pavinduLakshan) [inline en/includes/guides/authentication/sso-integrations/add-zoom-template.md:8]:
```suggestion
## Create the Zoom Application
```

**Comment by himeshsiriwardana** [comment]
Please refer the modified Microsoft 365 documentation below:

screencapture-localhost-8000-en-next-guides-authentication-sso-integrations-add-microsoft-365-template-2024-12-10-11_55_46

File: en/includes/guides/authentication/sso-integrations/add-zoom-template.md (hunk 8 above)
Comment (himeshsiriwardana) [reply en/includes/guides/authentication/sso-integrations/add-zoom-template.md]: Fixed with https://github.com/wso2/docs-is/pull/4893/commits/32b3f3ec55479adc8d3da2b7042bd292a7b20ce1

**Review by himeshsiriwardana — APPROVED** [review]

**Pull request merged**